src/participant.rs

```diff
@@ -126,6 +128,13 @@ impl Identity {
             .verify(&authenticated_data, &self.signature)
     }
 
+    #[must_use]
+    pub fn to_frost_identity(&self) -> frost::Identifier {
+        frost::JubjubBlake2b512::HID(&self.serialize()).expect("HID failed");
```

Comment:
What is this line trying to do, seems like it is expecting an error? The parameter is not mutable either so it seems like it isn't changing the application state at all?

Reply:
Thanks for spotting this; removed.

That was a leftover debug line: I initially added a dependency on `reddsa` from crates.io, and could not understand why the code was not working (this code only works with the unreleased version of `reddsa`).